addon/plugins/article-structure-plugin/structures/structure-header-number.ts

```diff
@@ -36,5 +39,18 @@ export const structure_header_number: NodeSpec = {
       },
       contentElement: getRdfaContentElement,
     },
+    // Backwards compatibility
+    {
+      tag: 'span',
+      getAttrs(node: HTMLElement) {
+        if (hasRDFaAttribute(node, 'property', ELI('number'))) {
+          return {
+            rdfaNodeType: 'literal',
+            property: ELI('number').prefixed,
```

Comment:
This isn't being pre-processed to have the new attrs such as backlinks, rdfaNodeType and __rdfaid, so `getRdfaAttrs` is returning `false`. I'm not sure why. Given that we intend to replace these nodespecs I didn't think it was worth digging into more. I also didn't want to have to replicate the logic of `getRdfaAttrs` to try to generate the correct attrs since we're mostly looking at backwards compatibility with these nodespecs.